### ISSUE TITLE
👨‍🌾 Fails to build when FastRTPS is enabled on Rolling

### ISSUE THREAD

**Issue by azeey** [ros2/performance_test#29]
Builds are failing on the Rolling nightly performance job (https://build.ros2.org/job/Rci__nightly-performance_ubuntu_focal_amd64/280/) with the error message
```
In file included from /tmp/ws2/src/performance_test/performance_test/src/communication_abstractions/resource_manager.cpp:15:
/tmp/ws2/src/performance_test/performance_test/src/communication_abstractions/resource_manager.hpp:19:12: fatal error: fastrtps/participant/Participant.h: No such file or directory
   19 |   #include <fastrtps/participant/Participant.h>
```

Locally I can see the build flags have changed from 
```
-I/tmp/ws/install_isolated/fastrtps/include -I/tmp/ws/install_isolated/fastrtps/include/fastrtps/include
```
to
```
-I/fastrtps/include
```

I think it's related to https://github.com/ros2/rosidl_typesupport_fastrtps/pull/68.

---
https://github.com/osrf/buildfarmer/issues/192

**Comment by tfoote** [comment]
I think you're right that it's not adding the fastrtps include path correctly. I think it's the other way though in need a non-prefixed include to find that file. 

```
root@6469c28231ca:/# locate Participant.h
/opt/ros/foxy/include/fastdds/dds/domain/DomainParticipant.hpp
/opt/ros/foxy/include/fastdds/rtps/participant/RTPSParticipant.h
/opt/ros/foxy/include/fastrtps/participant/Participant.h
/opt/ros/foxy/include/fastrtps/rtps/participant/RTPSParticipant.h
```

**Comment by clalancette** [comment]
This was fixed by #30, so closing this.
